docs/UserGuide.md

```diff
@@ -150,556 +157,1347 @@ An example of the final command is displayed below.
+
+<div markdown="block" class="alert alert-tip">
+
+:information_source: **Information**
+
+* Statuses available for setting are
+  * `N` or `n` for `Not Done`
+  * `D` or `d` for `Done`
+* If no status is provided, it is **defaulted** to `Not Done`.
+
+</div>
+
+<div markdown="block" class="alert alert-example">
+
+:clipboard: **Example 1: Standard command without status**
 
-* Status available for setting are: `N` / `n` (Not done), `D` / `d` (Done).
-* If no status is provided, it is defaulted to `N` (Not done).
+* `add_task n/Buy cookie cutter d/22/12/2024`<br>
+  Adds a task with the following details:
+  * Task Name: `Buy cookie cutter`
+  * Deadline: `22/12/2024`
+  * Status: `Not Done` (Default)
 
-Examples:
+:clipboard: **Example 2: Shortcut command with status**
 
-* `add_task n/Buy cookie cutter d/2022-12-22`
-* `add_t n/Buy a card d/2023-12-23 s/D`
+* `add_t n/Buy a card d/23/12/2024 s/D`<br>
+  Adds a task with the following details:
+  * Task Name: `Buy a card`
+  * Deadline: `23/12/2024`
+  * Status: `Done`
 
-### Adding a menu item: `add_item`
+</div>
+
+### 2.1.4 Adding a menu item: `add_item`
 
 Adds a menu item to the menu.
 
 Syntax: `add_item n/ITEM_NAME pr/PRICE c/COST`
 
-* When entering price and cost data, you may only
-  * input positive numbers.
-    * For example, you can input the number 5, not -5.
-  * choose to include number with a decimal point. A decimal point looks like this: `.` However, you can only key in up to 2 numbers after the decimal point.
-    * For example, the number 5.75 but not 5.752.
-* After you have add an item, note that the profit margin displayed can be positive or negative number.
-  * For example, if you see `Profit Margin: $-0.50` and  `Profit Margin: $2.00`, both are correct. However, this means that you are making a lost on your menu item.
+<div markdown="block" class="alert alert-tip">
+
+:information_source: **Information**
+
+* When entering price and cost values, you may choose to
+  1. input **positive** integer numbers
+  2. input **positive** decimal numbers (up to 2 decimal point). For example:<br>
+     :heavy_check_mark: `5.1` or `5.03`<br>
+     :x: `5.034`
+  
+* The item can have a positive or negative profit margin (Profit Margin = Price - Cost). Specifically:
+  * Negative profit margin means you are making a **loss**
+  * Positive profit margin means you are making a **profit**
+
+</div>
+
+<div markdown="block" class="alert alert-example">
+
+:clipboard: **Example 1: Standard command with integer values**
+
+* `add_item n/Chocolate cake pr/5 c/2`<br>
+  Adds an order item with the following details:
+  * Item Name: `Chocolate cake`
+  * Price: `5`
+  * Cost: `2`
+
+:clipboard: **Example 2: Shortcut command with decimal values**
+
+* `add_i n/Limited Time Strawberry Cake pr/20.55 c/25.2`<br>
+  Adds an order item with the following details:
+  * Item Name: `Limited Time Strawberry Cake`
+  * Price: `20.55`
+  * Cost: `25.2`
```

Comment:
Would `5.00`,  `2.00` and `25.20` be better? Since it is how it will be reflected in the Ui